rubocop.yml

```diff
@@ -13,9 +13,12 @@ Lint/AmbiguousOperator:
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 Lint/AssignmentInCondition:
-  AllowSafeAssignment: false
+  AllowSafeAssignment: true
 Lint/HandleExceptions:
   Enabled: false
+Metrics/AbcSize:
+  Enabled: true
+  Max: 25
```

Comment:
On the last part I agree somewhat: humans are good at fixing it but bad at spotting it - hence the metric.